### PR TITLE
Adding checksum links to README.md

### LINKED ISSUE #4532: Display checksums for installation artifacts in README
The checksums for the CLI installation artifacts should be displayed in the README, probably next to the Download links.  The checksums are located in the `dotnetclichecksums` storage account, with ".sha" appended to the filename; for example, the checksum for https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-win-x64.latest.exe is at https://dotnetclichecksums.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-win-x64.latest.exe.sha.

The checksums should also be included in the download page (https://www.microsoft.com/net/download#core).

Note that core-setup checksums are not yet produced; that work is tracked in https://github.com/dotnet/core-setup/issues/279.

### PR DESCRIPTION
**Pull request by blackdwarf**
Adding a link next to the SDK installer and binary download to link to checksums.

Fixes #4532 

/cc @livarcocc @naamunds 

skip ci please

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,18 +49,18 @@ You can download .NET Core as either an installer (MSI, PKG) or a zip (zip, gzip
...
+| **Windows x64** | [![](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/Windows_x64_Release_version_badge.svg)](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/latest.version) | [Download](https://dotnetcli.blob.core.windows.net/dotnet/preview/Installers/Latest/dotnet-win-x64.latest.exe) | [Download](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-win-x64.latest.exe) [Checksums](https://dotnetclichecksums.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-win-x64.latest.exe.sha) | [Download](https://dotnetcli.blob.core.windows.net/dotnet/preview/Binaries/Latest/dotnet-win-x64.latest.zip) | [Download](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-win-x64.latest.zip) [Checksums](https://dotnetclichecksums.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-win-x64.latest.zip.sha) |
```
Comment (naamunds) [inline README.md]: Consider naming the links "Checksum" instead of "Checksums", because there's only one for each artifact.

File: README.md [hunk 2]
```diff
@@ -49,18 +49,18 @@ You can download .NET Core as either an installer (MSI, PKG) or a zip (zip, gzip
...
+| **openSUSE 13.2** | [![](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/openSUSE_13_2_x64_Release_version_badge.svg)](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/latest.version) | N/A | N/A | [Download](https://dotnetcli.blob.core.windows.net/dotnet/preview/Binaries/Latest/dotnet-opensuse.13.2-x64.latest.tar.gz) | [Download](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-opensuse.13.2-x64.latest.tar.gz) [Checksums](https://dotnetclichecksums.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-opensuse.13.2-x64.latest.tar.gz.sha) |
+| **Fedora 23** | [![](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/Fedora_23_x64_Release_version_badge.svg)](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/latest.version) | N/A | N/A | [Download](https://dotnetcli.blob.core.windows.net/dotnet/preview/Binaries/Latest/dotnet-fedora.23-x64.latest.tar.gz) | [Download](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-fedora.23-x64.latest.tar.gz) [Checksums](https://dotnetclichecksums.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-fedora.23-x64.latest.tar.gz.sha) |
```
Comment (naamunds) [inline README.md]: Will these also be added to https://www.microsoft.com/net/download/core?

**Comment by piotrpMSFT** [comment]
@blackdwarf are you planning to open PRs to each branch to point to the checksums for them as well?

File: README.md (hunk 2 above)
Comment (blackdwarf) [inline README.md]: @naamunds yes, but I need to think it through. :)

File: README.md (hunk 1 above)
Comment (blackdwarf) [inline README.md]: @naamunds good point.

**Comment by blackdwarf** [comment]
@piotrpMSFT I would, but all of the point to the latest with the installers as well, if I'm not mistaken. If I am, will do it :)

**Comment by piotrpMSFT** [comment]
@mattgertz

**Pull request merged**